### ISSUE TITLE
Generated ID "Lighting5/IT" results in invalid device HASS  

### ISSUE THREAD

**Issue by Sundown89** [Sennevds/RFXcomIdGenerator#5]
Hi,
When I transmit an ON command with RFXmngr with random ID/unit my IT(ProMax) plug switches on correctly when in learning mode. After this I'm able to switch on/off using RFXmngr. However, when I use these ID/unit codes to generate an ID for HASS, HASS won't accept it. "Invalid device" is the error.

ID: 01 01
Unit code: A
Level: 1

Tried:
`python generateId.py -t Lighting5 -s ProMax -p 257,A` (decimal)
Result: 0a140f00011A0000
`python generateId.py -t Lighting5 -s ProMax -p 257,1` (decimal)
Result: 0a140f0001110000
`python generateId.py -t Lighting5 -s ProMax -p 0101,A` (hex)
Result: 0a140f0000101A0000
`python generateId.py -t Lighting5 -s ProMax -p 0101,1` (hex)
Result: 0a140f000010110000

Tried with several random ID/unit codes.

Can you tell me if this is a problem with the generator of within HASS? Running 0.72.1.

**Comment by Sennevds** [comment]
hi, I don't have a Lightning 5 device but had the same problem with ID's that worked in previous versions of hass but won't recognize in the latest version(can't remember when this started).
I think something changed in the original library. Will see if I get some time to check out what's the problem

**Comment by robvanuden** [comment]
Have you ever found a solution for this?

**Comment by Sundown89** [comment]
No, I did not. I in fact stopped using 433mhz devices and switched over to Zigbee (Xiaomi/IKEA) and wifi/mqtt (Sonoff).

**Comment by robvanuden** [comment]
I found an ID which is working for IT/ProMax: 
`0a140f01000b5501010f60`
